packages/visualizations/src/components/Table/Pagination/Pagination.svelte

```diff
@@ -0,0 +1,55 @@
+<script lang="ts">
+    import Pages from './Pages.svelte';
+    import Numbering from './Numbering.svelte';
+    // import PerPage from "./PerPage.svelte";
+    import type { TableOptions } from '../types';
+
+    export let onChangePage: Required<TableOptions>['pages']['onChangePage'];
+    export let totalRecords: number;
+    export let recordsPerPage: number;
+    export let initial: number;
+
+    let current = initial;
+
+    const setPage = async (page: number) => {
+        onChangePage(page);
+        current = page;
+    };
+
+    // "floor" because it's ceil but starts at 0
+    const totalPages = Math.floor(totalRecords / recordsPerPage);
```

Comment:
If totalRecords not a multiple of recordsPerPage, we are missing a page. 

15/10 => 1.5 => 1 (with Math.floor)
We need two pages (first to display the first ten records and a second one to display the last five records